.github/workflows/verify-fleetd-base.yml

```diff
@@ -64,6 +64,34 @@ jobs:
           curl -o fleetd-base-permalink.pkg "$(jq --raw-output '.fleetd_base_pkg_url' meta.json)"
           diff fleetd-base.pkg fleetd-base-permalink.pkg
 
+      - name: Generate summary of errors
+        if: failure()
+        run: |
+          c1grep() { grep "$@" || test $? = 1; }
+          c1grep -oP 'FAIL: .*$' /tmp/gotest.log > /tmp/summary.txt
+          c1grep 'test timed out after' /tmp/gotest.log >> /tmp/summary.txt
+          c1grep 'fatal error:' /tmp/gotest.log >> /tmp/summary.txt
+          c1grep -A 10 'panic: runtime error: ' /tmp/gotest.log >> /tmp/summary.txt
+          c1grep ' FAIL\t' /tmp/gotest.log >> /tmp/summary.txt
+          GO_FAIL_SUMMARY=$(head -n 5 /tmp/summary.txt | sed ':a;N;$!ba;s/\n/\\n/g')
+          echo "GO_FAIL_SUMMARY=$GO_FAIL_SUMMARY"
+          if [[ -z "$GO_FAIL_SUMMARY" ]]; then
+            GO_FAIL_SUMMARY="unknown, please check the build URL"
+          fi
+          GO_FAIL_SUMMARY=$GO_FAIL_SUMMARY envsubst < .github/workflows/config/slack_payload_template.json > ./payload.json
+
+      - name: Slack Notification
+        if: github.event.schedule == '0 4 * * *' && failure()
```

Comment:
@lucasmrod, is the first part of this condition necessary here (and below)?

Reply:
figuring not, since we want this to alert whenever it fails, not just at 4 in the morning